src/components/Details.jsx

```diff
@@ -0,0 +1,85 @@
+import { useParams, Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { AiOutlineSetting } from 'react-icons/ai';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import { IoChevronBackCircleOutline } from 'react-icons/io5';
+import { fetchTrips } from '../redux/tripSlice';
+import Loader from './Loader';
+import '../assets/stylesheets/Details.css';
+
+// the commented code is for future use
+
+export default function Details() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const Details= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.


![image](https://user-images.githubusercontent.com/57421823/224182045-fce402a1-ebfd-4e31-bc39-ffc699ff3eea.png)